ee/app/livechat-enterprise/server/lib/Department.js

```diff
@@ -30,3 +31,22 @@ export const findAllDepartmentsByUnit = async (unitId, offset, count) => {
 
 	return { departments, total };
 };
+
+export const findAllDepartmentsByTag = async (tagId) => {
+	const tag = LivechatTag.findOneById(tagId);
+
+	const departments = [];
+
+	if (!tag) {
+		return { departments };
+	}
+
+	if (tag.departments && tag.departments.length) {
+		tag.departments.forEach((departmentId) => {
+			const department = Promise.await(LivechatDepartment.findOneById(departmentId));
```

Comment:
Is it possible to do this query just once? Like with a `findByIds` method on `LivechatDepartment`.